climada/hazard/base.py

```diff
@@ -1326,7 +1325,7 @@ def _read_att_mat(self, data, file_name, var_names):
         self.frequency = np.squeeze(data[var_names['var_name']['freq']])
         self.orig = np.squeeze(data[var_names['var_name']['orig']]).astype(bool)
         self.event_id = np.squeeze(
-            data[var_names['var_name']['even_id']].astype(np.int, copy=False))
+            data[var_names['var_name']['even_id']].astype(int, copy=False))
```

Comment:
See https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated